samples/place-reviews/index.ts

```diff
@@ -0,0 +1,99 @@
+/**
+ * @license
+ * Copyright 2025 Google LLC. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+// [START maps_place_reviews]
+let innerMap;
+let infoWindow;
+const mapElement = document.querySelector('gmp-map') as google.maps.MapElement;
+
+async function initMap() {
+    // Import the needed libraries.
+    const [{ InfoWindow }, { AdvancedMarkerElement }, { Place }] =
+        await Promise.all([
+            google.maps.importLibrary(
+                'maps'
+            ) as Promise<google.maps.MapsLibrary>,
+            google.maps.importLibrary(
+                'marker'
+            ) as Promise<google.maps.MarkerLibrary>,
+            google.maps.importLibrary(
+                'places'
+            ) as Promise<google.maps.PlacesLibrary>,
+        ]);
+
+    innerMap = mapElement.innerMap;
+
+    // [START maps_place_reviews_get_first]
+    // Create a new Place instance.
+    const place = new Place({
+        id: 'ChIJpyiwa4Zw44kRBQSGWKv4wgA', // Faneuil Hall Marketplace, Boston, MA
+    });
+
+    // Call fetchFields, passing 'reviews' and other needed fields.
+    await place.fetchFields({
+        fields: ['displayName', 'formattedAddress', 'location', 'reviews'],
+    });
+
+    // Create an HTML container.
+    const content = document.createElement('div');
+    const titleDiv = document.createElement('div');
+    const ratingDiv = document.createElement('div');
+    const addressDiv = document.createElement('div');
+    const reviewDiv = document.createElement('div');
```

Comment:
nit: Probably can remove the "Div" suffixes

```suggestion
    const title = document.createElement('div');
    const rating = document.createElement('div');
    const address = document.createElement('div');
    const review = document.createElement('div');
```